test/v1/controllers/requestParsers/validators/DeleteReliefInvestmentsValidatorSpec.scala

```diff
@@ -48,6 +48,11 @@ class DeleteReliefInvestmentsValidatorSpec extends UnitSpec {
         validator.validate(DeleteReliefInvestmentsRawData(validNino, "2019-21")) shouldBe List(RuleTaxYearRangeInvalidError)
       }
     }
+    "return RuleTaxYearNotSupportedError" when {
+      "the tax year range exceeds 1" in {
```

Comment:
Currently says "the tax year range exceeds 1" for `RuleTaxYearNotSupportedError`